docs/sphinx/reference-frontend-api.rst

```diff
@@ -595,3 +599,27 @@ Functions
 .. function:: void obs_frontend_reset_video(void)
 
    Reloads the UI canvas and resets libobs video with latest data from profile.
+
+---------------------------------------
+
+.. function:: void obs_frontend_set_tbar_position(int position)
+
+   Sets the value of the transition bar.
+
+   :param position: The position to set the t-bar to
+   :value Range: * 0 - 1023
+
+---------------------------------------
+
+.. function:: void obs_frontend_release_tbar(void);
+
+   Release the transition bar.   (like a user releasing their mouse button after moving it).
+
+---------------------------------------
+
+.. function:: int obs_frontend_get_tbar_value(void)
+
+   Gets the value of the transition bar.
+
+   :return: value of the tbar position
+   :value Range: 0 - 1023
```

Comment:
Similar to above, I think this should be:
```
:return: The value of the position of the T-bar to, with a value in 0-1023.
:rtype: int
```

The `:value` syntax appears to be meant for ":value: initial value of the variable (text)" rather than possible values.  Also, `:value` and `:return` seem to be more focused on Python than C/C++, but perhaps that a problem for another day.

https://www.sphinx-doc.org/en/3.x/usage/restructuredtext/domains.html#info-field-lists